docs/creating_component_code_review.md

```diff
@@ -11,6 +11,8 @@ A checklist of things to do when you're adding a new component.
  1. Follow our [Style guidelines](development_guidelines.md)
  2. Use existing constants from [`const.py`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/const.py)
     * Only add new constants to `const.py` if they are widely used. Otherwise keep them on components level
+ 3. Add yourself to [`CODEOWNERS`](https://github.com/home-assistant/home-assistant/blob/dev/CODEOWNERS) so you [automatically get requested to review](https://help.github.com/en/articles/about-code-owners#codeowners-syntax) the code when it changes in the future
```

Comment:
Should this be expected going forward? By not signing a code owner that has signalled to core this is a less prioritised integration and will probably be removed if it breaks

Reply:
I don't think we should encourage every drive-by contributor to become a code owner.

Reply:
CODEOWNERS are only notified if members of the organization. 

I would remove this line.

Reply:
Ah didn't know it only worked for members.  Removed.